src/applications/check-in/day-of/containers/withSession.jsx

```diff
@@ -20,7 +16,11 @@ const withSession = Component => {
     const { router, setAuthenticatedSession, setToken } = props;
     const selectCheckInData = useMemo(makeSelectCheckInData, []);
     const checkInData = useSelector(selectCheckInData);
-
+    const {
```

Comment:
Just because this file was touched, We should update the loading component to be the web-component instead